Sources/BraintreeVenmo/Public/BraintreeVenmo/BTVenmoRequest.h

```diff
@@ -6,10 +6,12 @@ NS_ASSUME_NONNULL_BEGIN
  Usage for the tokenized Venmo account: either multi-use or single use
```

Comment:
```suggestion
 Usage for the resulting payment method
```
I'm not sure if we actually need to say what the cases are here. But if we do, we should probably list the new case as well.